css/common.css

```diff
@@ -0,0 +1,254 @@
+     *,
+     *::before,
+     *::after {
+         box-sizing: border-box;
+         margin: 0;
+         padding: 0;
+         border: 0;
+     }
+     
+      :root {
+         /* ### Colors */
+         --clr-dark-beige: #E6DACD;
+         --clr-light-beige: #F4ECE6;
+         --clr-royal-blue: #0150FD;
+         --clr-black: #000;
+         --clr-white: #FFF;
+         /* ### Fonts-Family */
+         --font-poppins: 'Poppins', sans-serif;
+         --font-quicksand: 'Quicksand', sans-serif;
+         /* ### Fonts Size */
+         --font-size--14px: 0.875rem;
+         --font-size--15px: 0.9375rem;
+         --font-size--16px: 1rem;
+         --font-size--17px: 1.0625rem;
+         --font-size--19px: 1.1875rem;
+         --font-size--20px: 1.25rem;
+         --font-size--21px: 1.3125rem;
+         --font-size--40px: 2.5rem;
+         /* ### Fonts Weight */
+         --font-weight--400: 400;
+         --font-weight--600: 600;
+         --font-weight--700: 700;
+     }
+     
+     body {
+         line-height: 1.5rem;
+         font-family: var(--font-quicksand);
+         font-size: var(--font-size--16px);
+         font-weight: var(--font-weight--400);
+     }
+     
+     @media screen and (min-width: 481px) {}
+     
+     @media screen and (min-width: 950px) {
```

Comment:
great that you've built the site mobile-first, and that you've separated CSS into smaller files! You can add media queries to any CSS file so that the media query sits alongside the mobile styles, for example if these classes are defined in the `header.css` file, you can add a media query in that file for the tablet & desktop header styles